### PR TITLE
417 android app crash in net9

### PR DESCRIPTION
**Pull request by amosekr**
.Net 9 apps we are getting compilation error in android specific code in the LabelHanlder, EntryHandler and EditorHandler classes.
Fixed those compilation error.

### CODE REVIEW

**Review by rthanga1 — APPROVED** [review]

File: Trimble.Modus.Components/Controls/BorderlessEditor/BorderlessEditor.cs [hunk 1]
```diff
@@ -3,6 +3,7 @@ namespace Trimble.Modus.Components;
 internal class BorderlessEditor : Editor
 {
     public BorderlessEditor()
-    {
+    {
+        BackgroundColor = Colors.Transparent;
```
Comment (rthanga1) [inline Trimble.Modus.Components/Controls/BorderlessEditor/BorderlessEditor.cs:7]: Is this also necessary?

File: Trimble.Modus.Components/Handlers/EditorHandler.cs [hunk 2]
```diff
@@ -22,7 +22,7 @@ public static void MapTMEditor(IEditorHandler editorHandler, IEditor editor)
 #if ANDROID
                 editorHandler.PlatformView.SetPadding(0, 0, 0, 0);
                 editorHandler.PlatformView.Background = null;
-                editorHandler.PlatformView.BackgroundTintList = Android.Content.Res.ColorStateList.ValueOf(Colors.Transparent.ToAndroid());
```
Comment (rthanga1) [inline Trimble.Modus.Components/Handlers/EditorHandler.cs:25]: Colors.Transparent is missing in Android?

File: Trimble.Modus.Components/Controls/BorderlessEditor/BorderlessEditor.cs (hunk 1 above)
Comment (amosekr) [reply Trimble.Modus.Components/Controls/BorderlessEditor/BorderlessEditor.cs:7]: Yes we need this for https://github.com/trimble-oss/modus-mobile-maui-components/issues/419.
In groundworks dark mode we are seeing two views in TMInput, to fix this i have added this line
image

File: Trimble.Modus.Components/Handlers/EditorHandler.cs (hunk 2 above)
Comment (amosekr) [reply Trimble.Modus.Components/Handlers/EditorHandler.cs:25]: Colors.Transparent.ToAndroid() this not support for .net 9 apps, hence i have changed to Android.Graphics.Color.Transparent

**Pull request merged**